string voice_filename
---
bool awake_flag
